scripts/run_example

```diff
@@ -12,7 +12,7 @@ serverargs=""
 while getopts "s:a:c:de:vh" opt; do
   case "${opt}" in
     h)
-      echo -e "Usage: ${0} [-h] [-s base|logless|none] [-a rust|cpp] [-c rust|cpp] [-d] [-v] -e EXAMPLE [-- CLIENT_ARGS]
+      echo -e "Usage: ${0} [-h] [-s base|logless|none] [-a rust|cpp|nodejs] [-c rust|cpp] [-d] [-v] -e EXAMPLE [-- CLIENT_ARGS]
```

Comment:
I think nodejs was added to the wrong flag?

Reply:
oh gosh yes, thanks for catching that. Careless mistakes are my common PR vulnerability